src/components/product.js

```diff
@@ -195,13 +188,11 @@ export default class Product extends Component {
 
   get carouselImages() {
     return this.model.images.map((image) => {
-      return Object.assign({}, image, {
-        // isSelected: image.id === this.currentImage.id,
-        isSelected: true,
-        // use correct image size
-        // carouselSrc: image.variants[3].src,
```

Comment:
A lot of this is no longer required because we now fetch resized images with `ShopifyBuy.Image.Helpers.imageForSize`, we we just have to keep track of a single image GraphModel for what's currently selected